etc/firebase-admin.firestore.api.md

```diff
@@ -100,11 +100,20 @@ export function getFirestore(): Firestore;
 // @public
 export function getFirestore(app: App): Firestore;
 
+// @beta
+export function getFirestore(databaseId: string): Firestore;
+
+// @beta
+export function getFirestore(app: App, databaseId: string): Firestore;
+

```

Comment:
I think if we combine the two methods the docs will be more readable.

```
export function getFirestore(app?: App, databaseId: string): Firestore;
```

Reply:
I think inviting customers to pass null as a parameter is a bad idea. 

The problem is in doc generation that doesn't include parameters in signature:

<img width="1143" alt="image" src="https://github.com/firebase/firebase-admin-node/assets/13738054/fffb72c9-ca1b-4d6c-8a5d-ec0386e5dfa2">

Reply:
What I meant to say was to keep both as optional parameters so you don't need three method signatures. It also simplifies the docs.

```
export function getFirestore(app?: App, databaseId?: string): Firestore;

export function getFirestore(
  appOrDatabaseId?: App | string,
  optionalDatabaseId?: string
): Firestore {
```

Reply:
I still think calling `getFirestore(null, 'myDB')` is worse than having overload `getFirestore('myDB')`.

TypeScript doesn't expose the last signature, nor would I want to. The signature suggests someone could write code `getFirestore('myDB', 'myDB')` which isn't allowed.

```
export function getFirestore(
  appOrDatabaseId?: App | string,
  optionalDatabaseId?: string
): Firestore
```